### DISCUSSION TITLE
push map

### DISCUSSION THREAD

**Discussion by rionda** [tpope/vim-fugitive#1903] [Q&A] [answered]
I may be missing it in the docs, but is there a `fugitive-map` to push committed changes from the `fugitive-summary`?
If not, is it a deliberate choice?
Thanks.

**Comment by tpope** [comment, marked as answer]
The staging map `s` has an undocumented behavior of generating a `:Git push` command if invoked with the cursor positioned on an unpushed commit. Long term I would like to add something more official, but there are more fundamental problems with the mapping system to solve first.

**Reply by rionda** [reply]
Thank you Tim!

**Reply by goldjacobe** [reply]
I know I'm resurrecting this almost two years later, but I'm curious what those problems are — is it something a contribution could help with?

**Comment by metan283** [comment]
در تاریخ سه‌شنبه ۷ دسامبر ۲۰۲۱،‏ ۱۷:۲۶ Matteo Riondato <
***@***.***> نوشت:
